backend/code_review_backend/issues/serializers.py

```diff
@@ -183,7 +183,7 @@ class IssueCheckStatsSerializer(serializers.Serializer):
 
     repository = serializers.CharField(source="diff__revision__repository__slug")
     analyzer = serializers.CharField()
-    check = serializers.CharField()
+    analyzer_check = serializers.CharField()
```

Comment:
By doing that, you change the API output and break the frontend: you should maintain `check` as name and specify the field source instead coming from `analyzer_check`